internal/driver/glfw/device_goxjs.go

```diff
@@ -0,0 +1,34 @@
+//go:build js || wasm
+// +build js wasm
+
+package glfw
+
+import (
+	"regexp"
+	"syscall/js"
+
+	"fyne.io/fyne/v2"
+)
+
+var AndroidId = regexp.MustCompile(`Android`)
+var BlackBerryID = regexp.MustCompile(`BlackBerry`)
+var iOSID = regexp.MustCompile(`iPhone|iPad|iPod`)
+var OperaID = regexp.MustCompile(`Opera Mini`)
+var WindowsMobileID = regexp.MustCompile(`IEMobile`)
+
+var navigator = js.Global().Get("navigator")
+var userAgent = navigator.Get("userAgent").String()
```

Comment:
I suspect that the return type of this is a byte slice. It's probably better to not convert it to a string and just use the `.Match()` method instead.

Reply:
I think it is actually a JS Value like all the API that cross to the JS world.

Reply:
Ah, yeah. You're probably right.